mainhandler/kubescapehandlerhelper_test.go

```
@@ -256,3 +258,21 @@ func TestAppendSecurityFramework(t *testing.T) {
+
+func TestReadKubescapeV1ScanResponseIncludesBodyOnError(t *testing.T) {
+	// A real *http.Response is required here, not a hand-built one: resp.Body's
+	// concrete type behind the io.ReadCloser interface matters for what %s used
+	// to print, and only a genuine transport-produced body reproduces that.
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusInternalServerError)
+		_, _ = w.Write([]byte("scan queue is full"))
+	}))
+	defer srv.Close()
+
+	resp, err := http.Get(srv.URL)
```

golangci-lint (2.12.2)

[error] net/http.Get must not be called. use net/http.NewRequestWithContext and (*net/http.Client).Do(*http.Request)

(noctx)